test/application/rooms.js

```diff
@@ -0,0 +1,25 @@
+var assert = require('assert');
+var room;
+
+describe('Rooms features', function () {
+	describe('Rooms', function () {
+		describe('Rooms.get', function () {
+			it('should be a function', function () {
```

Comment:
I'd rather do it in idiomatic Mocha. Being immediately accessible to someone familiar with Mocha (or have read the Mocha docs) is more important to me than not having dumb function names.